src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+import java.util.List;
+
+public class Report {
```

Comment:
Same. I don't think the countdown should be an object. I think you should change your logic in that direction.